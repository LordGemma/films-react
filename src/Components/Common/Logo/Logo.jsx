import React from "react";
import "./Logo.scss";

export const Logo = ({ title }) => {
  return (
    <div className="logo">
      <p>{title || "netflixroulette"}</p>
    </div>
  );
};
