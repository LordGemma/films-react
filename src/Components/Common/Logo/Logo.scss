.logo {
    margin: 15px;
    color: #ea445d;
    p {
        margin: 0;
        font-weight: bold;
    }
}