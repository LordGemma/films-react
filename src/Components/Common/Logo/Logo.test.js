import React from "react";
import { shallow } from "enzyme";
import { Logo } from './Logo';

describe('Logo component', () => {
    it('should render', () => {
        const wrapper = shallow(<Logo />);

        expect(wrapper).toMatchSnapshot();
    });

    it('should contain default text', () => {
        const wrapper = shallow(<Logo />);

        expect(wrapper.find("p").text()).toEqual("netflixroulette");
    })

    it('should contain passed text', () => {
        const wrapper = shallow(<Logo title='test logo' />);

        expect(wrapper.find("p").text()).toEqual("test logo");
    })
})