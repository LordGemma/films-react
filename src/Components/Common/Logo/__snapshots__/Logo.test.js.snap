// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render 1`] = `ShallowWrapper {}`;
